src/main/scala/org/apache/mesos/chronos/scheduler/config/StatsPublisherConfiguration.scala

```diff
@@ -0,0 +1,14 @@
+package org.apache.mesos.chronos.scheduler.config
+
+import org.rogach.scallop.ScallopConf
+
+trait StatsPublisherConfiguration extends ScallopConf{
+  lazy val webhookUrl = opt[String]("stats_webhook_url",
+    descr = "Url to which tasks statistics are pushed",
```

Comment:
This is a good point. In that case, I have to extend the configuration for credentials. But would it make sense to include that in this PR? Extending it for all the possible stores would be diffucult. The initial idea was that we send the data to the hook and this hook has the logic to do whatever it wants. That would be a nice seperation from the chronos as well. I do agree, however, that there should be a way to provide credential to push to the hook.